rpcs3/keyboard_pad_handler.cpp

```diff
@@ -174,42 +195,119 @@ std::vector<std::string> keyboard_pad_handler::ListDevices()
 	return list_devices;
 }
 
+const std::string keyboard_pad_handler::GetKeyName(const QKeyEvent* keyEvent)
```

Comment:
returning `const` primitive values (`u32`, `double` and etc.) doesn't make any difference and returning `const std::string` might even make code slower. See this [SO question](https://stackoverflow.com/questions/8716330/purpose-of-returning-by-const-value).